cart.spec

```diff
@@ -112,13 +112,18 @@ ln %{?buildroot}%{carthome}/{TESTING/.build_vars,.build_vars-Linux}.sh
 
 %files devel
 %{_includedir}/*
+%{_libdir}/libcart.so
+%{_libdir}/libgurt.so
```

Comment:
Good.  But you also need to exclude them from the main RPM's `%files` section above at line 108:
https://github.com/daos-stack/cart/blob/e69eaffa89602fb94171f6932d395f11388d3ca4/cart.spec#L108

So you can either change that very broad matching glob into some more narrow matching globs, to pick up everything except the `.so` files or you can use `%exclude`.

A lot of times it's simply good enough to do:
```
%{_libdir}/*.so.*
```